modules/builds-identifying-image-change-triggers.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * builds/triggering-builds-build-hooks.adoc
+
+[id="builds-image-change-trigger-identification_{context}"]
+= Image change trigger identification
+
+As a developer, if you have configured image change triggers, you can identify which image change initiated the last build.
+
+To accomplish this, you must identify elements in your build configuration's specification and status that are related to image change triggers.
+
+This way, you can use the timestamp in `buildConfig.status.imageChangeTriggers` to identify the most recent build. Then you can use the name and namespace of the image stream that triggered this build to find the corresponding image change trigger in the `buildConfig.spec.triggers`.
```

Comment:
You use the timestamp in `buildConfig.status.imageChangeTriggers` to identify the most recent build.  Then you use the name and namespace of the image stream that triggered the build to find the corresponding image change trigger in the `buildConfig.spec.triggers`.  [file?]

Reply:
Thanks, Julie. Updated.